javascript/0027-remove-element.js

```diff
@@ -39,6 +39,7 @@ var removeElement = function(nums, val) {
 
 
 /**
+ * Time O(N²) | Space O(1)
```

Comment:
I'm not sure, when I checked the v8 source code, it looks like .splice uses and argument called insertIndex, which could avoid the looping